src/core/styles/common/mixins/fonts.ts

```diff
@@ -26,6 +26,29 @@ export const fontBodyXs = fontBody("xs");
 export const fontBodyXxs = fontBody("xxs");
 export const fontBodyXxxs = fontBody("xxxs");
 
+type FontCapsSize = keyof Typography["styles"]["caps"];
+
+export const fontCaps = (fontSize: FontCapsSize) => {
+  return (props: Props): SerializedStyles | null => {
+    const typography = getTypography(props);
+
+    if (!typography) return null;
+
+    const {
+      styles: { caps },
+    } = typography;
+
+    return css`
+      ${themeToCss(caps[fontSize])}
+      text-transform: uppercase;
+    `;
+  };
+};
+
+export const fontCapsXxs = fontCaps("xxs");
```

Comment:
Chip uses this mixin, so might as well export them

Reply:
Nice! I end up using this for `TooltipTable` as well, so thanks for adding! 😄

Reply:
Amazing!! Go team 🚀 !!